src/legacy/core_plugins/dashboard_embeddable_container/public/shim/index.ts

```diff
@@ -18,6 +18,7 @@
  */
 
 import { PluginInitializerContext } from 'kibana/public';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { npSetup, npStart } from 'ui/new_platform';
```

Comment:
should we allow importing from `ui/new_platform`? if yes you can add `!ui/new_platform` in zone declaration inside .eslintrc